packages/core/content-manager/server/tests/relations-dp.test.e2e.js

```diff
@@ -0,0 +1,260 @@
+'use strict';
```

Comment:
2 general comments here:

- Could you add some error testing (validations, missing entity, ...)
- We could consider using a describe each to test both d&p et non d&p at the same time to make those 2 test files easier to maintain